content/en/agent/guide/agent-status-page.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Agent Status Page
+kind: guide
+further_reading:
+- link: "agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Agent Troubleshooting"
+- link: "agent/guide/agent-configuration-files/"
+  tag: "Guide"
+  text: "Agent Configuration Files"
+- link: "agent/guide/agent-commands/"
+  tag: "Guide"
+  text: "Agent Commands"
+---
+
+The Agent status page displays information about your running Agent. See [Agent Commands][1] to find the status command for your environment. The sections below provide details on the contents of the status page.
```

Comment:
Should we say that agent5 `info` page is completely different ?

Reply:
Changing to specify this is for Agent v6.